### ISSUE TITLE
Add a starting message

### ISSUE THREAD

**Issue by lynnstarke** [ID-Start-Winter22/chat-team-11#49]
As a chatbot user I want to receive a message when I open the chat widget. 

Expected outcome:
"Ich bin Travy - dein Zugreise-Assistent!
Gib mir deinen Zugnamen (zB. ICE 123) und ich informiere dich während deiner Zugreise über wichtige Informationen und Änderungen."

**Comment by lstarke1** [comment]
@theopfr @cma5 habe mir gerade überlegt, wenn wir die automated starting message nehmen: 

"Ich bin Travy - dein Zugreise-Assistent!
Gib mir deinen Zugnamen (zB. ICE 123) und ich informiere dich während deiner Zugreise über wichtige Informationen und Änderungen."

und der User danach "hallo" sagen würde, wäre es  doch besser wenn der Chatbot dann z.B. einfach "hallo" oder "hallo, wie geht's dir?" antwortet, anstatt wieder die selbe Nachricht. Sagt mir gerne eure Meinung :)

**Comment by lynnstarke** [comment]
@theopfr @cma5 wenn wir als automated starting message diese nehmen:

"Ich bin Travy - dein Zugreise-Assistent!
Gib mir deinen Zugnamen (zB. ICE 123) und ich informiere dich während deiner Zugreise über wichtige Informationen und Änderungen."

und der Chatbot User danach "hallo" sagt, wäre es doch besser wenn der Chatbot dann z.B. "hallo" oder "hallo, wie geht es dir" oder ähnliches sagt. Sagt mir gerne eure Meinung dazu :)

**Comment by theopfr** [comment]
@lynnstarke stimmt, wenn mans auf WhatsApp nutzen würde dann bräuchte man die alte Hallo Nachricht wahrscheinlich noch, weil der Bot nicht die Convo starten kann. Ich würde die nachricht erstmal so drin lassen und wenn es sich dann doch irgendwie goofy anhört ändern wir die antwort.

Und was wir/du auch noch beachten müssen ist, dass wenn der Nutzer zurück "hallo" sagt, dass der Bot dann sowas sagt wie "Wie kann ich dir helfen?"

**Comment by theopfr** [comment]
also ich hab viel ausprobiert und das scheint mit unserem Chat Widget nicht möglich zu sein, außer wir switchen von sockets auf die REST API

**Comment by theopfr** [comment]
so its done guys
